rest_framework_jwt/utils.py

```diff
@@ -23,9 +23,12 @@ def jwt_get_secret_key(payload=None):
     """
     if api_settings.JWT_GET_USER_SECRET_KEY:
         User = get_user_model()  # noqa: N806
-        user = User.objects.get(pk=payload.get('user_id'))
-        key = str(api_settings.JWT_GET_USER_SECRET_KEY(user))
-        return key
+        try:
+            user = User.objects.get(pk=payload.get('user_id'))
+            key = str(api_settings.JWT_GET_USER_SECRET_KEY(user))
+            return key
+        except User.DoesNotExist:
+            pass
```

Comment:
`jwt.InvalidTokenError()` exception should be raised, for sure there is not place for `pass` statement. Please take a look into the `BaseJSONWebTokenAuthentication` class where `InvalidTokenError` exception is handled.